packages/orchestration/src/typeGuards.js

```diff
@@ -245,6 +245,7 @@ export const ForwardOptsShape = M.splitRecord(
   {
     timeout: M.string(),
     retries: M.number(),
+    intermediateRecipient: M.string(),
```

Comment:
again, my brain goes: not `ChainAddressShape`?
I can see that it's correct, but using the same name for 2 things is hurting my brain.

Reply:
Fixed